conf/etc/v2/alert_event_rules.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "2.0.0",
+  "alert": {
+    "components": [
+      "ha"
```

Comment:
Do we need component ? Its always going to be 'ha' since we are doing development only for HA

Reply:
Yeahh I kept it for similarity between other rules json and this rule json.

Reply:
okay